cognite/v05/timeseries.py

```diff
@@ -155,7 +156,7 @@ def _get_datapoints_helper(name, aggregates=None, granularity=None, start=None,
     Returns:
         list of datapoints: A list containing datapoint dicts.
     """
-    api_key, project = kwargs.get("api_key"), kwargs.get("project")
+    api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))
```

Comment:
Is this necessary/correct? User should not use this method directly, and its caller method has already checked for global config.

Reply:
I mean get_datapoints as the caller method. The user should use configure_session to set client-wide configuration. Should not have to pass it in every request.

Reply:
Yes, I also agree that the end user should use `configure_session` if they only work on one project at the time and then it is not necessary to pass `api_key` and `project` in every request such at `get_datapoints`. But if you for some reason work with more that one project and/or api_key in one session, you need to have the ability to pass these argument in methods.

My objection was just that almost all the helper methods/private methods calls `api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))` even though the caller methods already do this. I changed this method to be more consistent with the other ones.

But I actually think the better approach would be to call `api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))` in the caller methods, then pass `api_key` and `project` to the helper methods and then call `api_key, project = kwargs.get("api_key"), kwargs.get("project")`. But then I think that the docstring in the helper methods should be 
```
        api_key (str):          Your api-key. Obligatory in this helper method.

        project (str):          Project name. Obligatory in this helper method.
```

Reply:
Agree. Great if you make the necessary changes and write tests!